analysis/frame-node.js

```diff
@@ -169,10 +169,11 @@ class FrameNode {
     const platformPath = getPlatformPath(systemInfo)
 
     const parentDir = platformPath.join(systemInfo.mainDirectory, `..${systemInfo.pathSeparator}`)
+    const typeTEMP = platformPath.relative(parentDir, platformPath.dirname(this.fileName))
 
     return {
       // TODO: use this type after adding custom d3-fg filter on properties including category
-      typeTEMP: platformPath.relative(parentDir, platformPath.dirname(this.fileName)),
```

Comment:
This change doesn't seem to do anything? Is it left over from debugging?

Doesn't really matter, but it'll cause a minor merge conflict in at least one open PR that removes typeTEMP

Reply:
Updated.